tests/platform_tests/verify_dut_health.py

```diff
@@ -134,7 +137,10 @@ def verify_dut_health(request, duthosts, rand_one_dut_hostname, tbinfo):
     check_services(duthost)
     check_interfaces_and_transceivers(duthost, request)
     check_neighbors(duthost, tbinfo)
-    pre_existing_cores = duthost.shell('ls /var/core/ | wc -l')['stdout']
+    if "20191130" in duthost.os_version :
+        pre_existing_cores = duthost.shell('ls /var/core/ | grep -v python | wc -l')['stdout']
```

Comment:
Can we add a comment here about why Python core is being ignored for 201911 images?

Reply:
The PR description has the details